### ISSUE TITLE
Missing clear() method in data types generated by yarpidl_rosmsg

### ISSUE THREAD

**Issue by randaz81** [robotology/yarp#1136]
It could be extremely useful in the following example:

```
ros_msg& data= m_rosPublisherPort.prepare();
data.clear(); //<----- this is missing!
data.id=5;
//...
m_rosPublisherPort.write();
```
 
Especially in the case when data contains vectors. 

NOTE: prepare() does not automatically clears the memory (it's like a buffered port).
The user is thus forced to clear all the fields of the data structure manually.

**Comment by drdanz** [comment]
Almost done, I have to do some testing though...

**Comment by drdanz** [comment]
Fixed by #1142
